index.bs

```diff
@@ -2702,11 +2702,14 @@ Determining the W3C Decision</h4>
 	those other conditions must also be re-fulfilled.
 
 	<div class="example">
-		For example, to make [=substantive changes=] to a [=Proposed Recommendations=],
-		the [=technical report=] could be returned to [=Candidate Recommendation=].
+		For example, to make [=substantive changes=] to a [=technical report=]
+		prior to publication as a [=Recommendation=],
+		it would need to go through a new [=Update Request=],
+		be republished as a new [=Candidate Recommendation=],
+		and fulfill the criteria for advancement.
```

Comment:
I find this example quite confusing to read now. I think there may be an unstated assumption that the technical report was a Rec before the substantive changes were made, and the goal is publication as a new Rec version?

Reply:
No, the assumption is that on a CR to REC transition, the Team cannot just change stuff an publish. Any normative change needs a WG decision, and be published as a CR first (so as to trigger patent exclusion opportunities).

I guess what's confusing here is that "those other conditions must also be re-fulfilled" is in practice not really something you can do in the middle of an AC Review wrap-up, so the realistic way to deal with this is to send the document back for further work, not to approved it with changes integrated (and extra steps to make that possible).

Reply:
Oh, okay. How about:

```suggestion
		For example, if [=substantive changes=] to a [=technical report=]
		are identified when assessing the transition from
		[=Candidate Recommendation=] to [=Recommendation=],
		the technical report would need to go through a new [=Update Request=],
		be republished as a new [=Candidate Recommendation=],
		and that new version would need to fulfill the criteria for advancement.
```